### DISCUSSION TITLE
Is it BitTheCat or CatTheBit

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [Schweinepriester/github-profile-achievements#154] [Q&A] [answered]
Is it BitTheCat or CatTheBit

**Comment by BitTheCat** [comment, marked as answer]
Maybe not ahah

**Reply by BitTheCat** [reply]
It's done, thx dude ^^

**Reply by BitTheCat** [reply]
@thenithinbalaji I received the bronze badge, so it's retro active

**Reply by thenithinbalaji** [reply]
hmm yeah

**Reply by BitTheCat** [reply]
@thenithinbalaji done ^^ If u wanna u can put star to my readme repository ^^
